classes/PodsInit.php

```diff
@@ -511,7 +520,7 @@ public function register_assets() {
 
 		if ( ! wp_script_is( 'jquery-ui-timepicker', 'registered' ) ) {
 			wp_register_script(
-				'jquery-ui-timepicker', PODS_URL . 'ui/js/timepicker/jquery-ui-timepicker-addon.min.js', array(
+				'jquery-ui-timepicker', PODS_URL . "ui/js/timepicker/jquery-ui-timepicker-addon{$suffix_min}.js", array(
```

Comment:
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: 6e7293be02d31887e0eb1615686c493e</sub>